project.owasp.yaml

```diff
@@ -0,0 +1,44 @@
+audience:
+  - breaker
+  - builder
+leaders:
+  - name: Elie Saad
+    email: elie.saad@owasp.org
+    github: ThunderSon
+  - name: Rick Mitchell
+    email: rick.mitchell@owasp.org
+    github: kingthorin
+community:
+  - platform: slack
+    url: https://app.slack.com/client/T04T40NHX/CJ2QDHLRJ
+    description: '#testing-guide on OWASP slack'
+logo:
+  - small: "https://github.com/OWASP/wstg/blob/master/.github/www/assets/202006-owasp_material/Github/Github%20-%20Shield%20400x400%20-%20Gradient%20%E2%80%93%201.png"
+    medium: "https://github.com/OWASP/wstg/blob/master/.github/www/assets/202006-owasp_material/Github/Github%20-%20Shield%20400x400%20-%20Gradient%20%E2%80%93%201.png"
+    large: "https://github.com/OWASP/wstg/blob/master/.github/www/assets/202006-owasp_material/Github/Github%20-%20Shield%20400x400%20-%20Gradient%20%E2%80%93%201.png"
```

Comment:
I mentioned this on slack but it's rather ridiculous to be required to specify all 3, I'd propose this should use `anyOf`

https://swagger.io/docs/specification/v3_0/data-models/oneof-anyof-allof-not/

Reply:
Added https://github.com/OWASP/nest-schema/issues/21